CHANGES.rst

```diff
@@ -34,6 +34,16 @@ jplhorizons
   Included in this update, the default reference system is changed from
   J2000 to ICRF, following API documentation. [#2154]
 
+- Query `id_type` behavior and options are changing:
```

Comment:
double backticks for all of these to make sure sphinx doesn't try to linkify them.

Reply:
Got it.